src/silo/prepare_dataset.cpp

```diff
@@ -295,15 +287,11 @@ std::unordered_map<std::string, silo::common::Date> sortMetadataFile(
 }
 
 void sortSequenceFile(
-   silo::FastaReader& sequence_in,
-   std::ostream& sequence_out,
+   silo::ZstdFastaReader& sequence_in,
+   silo::ZstdFastaWriter& sequence_out,
    const PartitionChunk& chunk,
```

Comment:
This seems unused. Is it really only used for logging? Then we could maybe log something else.